python/composio/tools/env/filemanager/manager.py

```diff
@@ -213,8 +215,11 @@ def grep(
         else:
             pattern = Path(pattern)
 
+        exclude_paths = [self.working_dir / Path(ex) for ex in (exclude or [])]
+
         results: t.Dict[str, t.List[t.Tuple[int, str]]] = {}
         paths_to_search: t.List[Path] = []
+        truncate_length = 1024
```

Comment:
The `truncate_length` variable seems like a configuration value. Consider moving it to a constant at the module or class level for better maintainability.